utils/windows_configuration.py

```diff
@@ -31,6 +31,15 @@ def get_registry_conf(agentConfig):
 
     return registry_conf
 
+def get_windows_sdk_check(name):
+    sdk_reg_path = "SOFTWARE\\Datadog\\Datadog Agent\\Integrations\\{}".format(name)
+    try:
+        with _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE,sdk_reg_path) as reg_key:
+            directory = _winreg.QueryValueEx(reg_key, "InstallPath")[0]
+            return os.path.join(directory, 'Integrations', name, 'check.py')
+    except WindowsError as e:
```

Comment:
that's the only time I would expect; others would be lack of proper permissions (which shouldn't happen because we're running as system), things like that.  If the OpenKey() fails, there's not much we can do but move on, anyway.  IMO this is fine.